src/callbacks.py

```diff
@@ -192,142 +192,17 @@ def on_epoch_end(self,
 
 class GradientVarianceCallback(EvaluationCallbackBase):
```

Comment:
Can you document this callback some more? E.g explain how you rely on qa pairs and definitions being ordered in a certain way, add comments where useful, etc